components/builder-jobsrv/src/migrations/00000000000000_diesel_initial_setup/down.sql

```diff
@@ -1,2 +0,0 @@
-DROP FUNCTION IF EXISTS diesel_manage_updated_at(_tbl regclass);
```

Comment:
Similar to above, drop these db items now or wait for a later time and a different PR. In this case this comment applies to all of the db related material in builder-jobsrv component.